code/modules/reagents/Chemistry-Goon-Medicine.dm

```diff
@@ -567,8 +567,8 @@ datum/reagent/epinephrine/on_mob_life(var/mob/living/M as mob)
 		M.adjustFireLoss(-1*REM)
 	if(M.oxyloss > 35)
 		M.setOxyLoss(35)
-	if(M.losebreath > 3)
-		M.losebreath = 3
+	if(M.losebreath >= 10)
```

Comment:
Wouldn't ```>``` make more sense?
I mean if it is already 10, then why change it to 10?

But I guess it's no huge deal.